### PR TITLE
Add Pancakeswap V2 for zkSync dex.trades_beta

### PR DESCRIPTION
**Pull request by lgingerich**
Still to do:
- Add seed file

### CODE REVIEW

**Comment by lgingerich** [comment]
I feel like whatever I'm doing wrong is a very obvious fix but I can't see it.

I can query `pancakeswap_v2_zksync.PancakePair_evt_Swap` just fine on Dune UI.

File: models/_sector/dex/trades/zksync/platforms/pancakeswap_v2_zksync_base_trades.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    )
+}}
+
+{{
+    uniswap_compatible_v2_trades(
+        blockchain = 'zksync',
+        project = 'pancakeswap',
+        version = '2',
+        Pair_evt_Swap = source('pancakeswap_v2_zksync', 'PancakePair_evt_Swap'),
+        Factory_evt_PairCreated = source('pancakeswap_v2_zksync', 'PancakeFactory_evt_PairCreated')
```
Comment (Hosuke) [inline models/_sector/dex/trades/zksync/platforms/pancakeswap_v2_zksync_base_trades.sql:19]: We just need to add these 2 sources in 
https://github.com/duneanalytics/spellbook/blob/main/sources/_sector/dex/trades/zksync/_sources.yml

File: models/_sector/dex/trades/zksync/_schema.yml [hunk 2]
```diff
@@ -19,3 +19,22 @@ models:
...
+    description: "Pancakeswap zkSync v2 base trades"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - evt_index
+      # - check_dex_base_trades_seed:
+      #     # seed_file: ref('pancakeswap_zksync_base_trades_seed')
+      #     filter:
+      #       version: 2
```
Comment (Hosuke) [inline models/_sector/dex/trades/zksync/_schema.yml:40]: I just added the sources.
You can continue adding the seed file and testing the spell.

**Comment by lgingerich** [comment]
`dex.pools` hasn't been setup at all in the new macro/sector structure, so there is quite a bit to do for it. I'll handle it in a separate PR to not hold this up.

**Comment by lgingerich** [comment]
Will fill the seed file next week

**Comment by Vtalike** [comment]
> Still to do:
> 
> - Add seed file
> 
> - Add dex.pools_beta

**Comment by lgingerich** [comment]
@Hosuke I'm kind of lost on this now. It was working, and all I did was update the seed file, and not it's failing on data type mismatch.

I was pretty sure I copied the format of other chains for everything so not sure what's going wrong.

File: seeds/_sector/dex/pancakeswap_zksync_base_trades_seed.csv
Comment (Hosuke) [inline seeds/_sector/dex/pancakeswap_zksync_base_trades_seed.csv:1]: You may miss the schema for this seed file.

Comment (lgingerich) [reply seeds/_sector/dex/pancakeswap_zksync_base_trades_seed.csv:1]: Thanks! Did not realize that was a thing

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @lgingerich 

[Sample pancakeswap_v2_zksync_base_trades]()

**Pull request merged**